doc/changelog.md

```diff
@@ -102,7 +102,7 @@ Detailed Notes
     Torch will unconditionally try to link in this library, however
     fails because the linking flags are incorrect.
     ([SmartSim-PR538](https://github.com/CrayLabs/SmartSim/pull/538))
--   Change type_extension and pydantic versions in readthedocs
+-   Change typing\_extensions and pydantic versions in readthedocs
```

Comment:
I notice two updates to the Detailed Notes section but one update to the Description!

Reply:
Yeah the change to "Description" and one of the "Detailed Notes" edits were just minor typo/grammar things.

The actual "Detailed Note" I added I figured fell under the category of "Minor enhancements to test suite" which is already listed in the "Description" section, so I just added something the details. LMK if you think this warrants its on entry in the "Description" and I can add something there!!